pkg/docker/containers-view.jsx

```diff
@@ -61,7 +61,7 @@ class Dropdown extends React.Component {
                     {
                         this.props.actions.map(function (action, index) {
                             return (
-                                <li key={index} className={ action.disabled ? 'disabled' : '' }>
+                                <li key={index} disabled={action.disabled}>
```

Comment:
Since disabled is invalid for li, we should not change this either.

Reply:
Indeed, can you please revert?